### ISSUE TITLE
A32NX will not automatically tune the ILS Frequency.

### ISSUE THREAD

**Issue by minkyusimmer** [flybywiresim/a32nx#6147]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-11-06T18:00:29+00:00",
  "ref": "master",
  "sha": "3fbce50cbd1d71f841aadbf19585f04628f97703",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

After I select the ILS Approach for KSMF (Sacramento International Airport) and insert it into the flight plan, the RADNAV page does not show any ILS frequency. I tried manually inputting from the radio page below the fmc, but clicking on NAV and then ILS, does not bring up the ILS radio page and the green light for the ILS button in the radio section is not illuminated. I also tried inputting the ILS freq info for runway 35L in KSMF in the RADNAV page in the MCDU (I specifically typed "IHUX/111.10" and pressed the line key for the "LS/Frequency" line, but the MCDU gave a message saying "NOT ALLOWED." Has anyone experience this problem?

### Expected behavior

After selecting the ILS Approach in the MCDU, the ILS frequency should populate in the RADNAV page in the MCDU.

### Steps to reproduce

1. Start-Up Aircraft at KSJC.
2. Enter THIS Flight Plan: KSJC/30R TECKY3 TECKY DCT SJC DCT BMRNG DCT ORRCA DCT KSMF/35L (note I selected the ILS Approach instead of going direct to KSMF).
3. Insert the Flight Plan with the ILS 35L approach selected. 
4. Check RADNAV page
5. Try manually inputting the ILS info on the RADNAV page and also try directly on the Radio Console below the fmc.

### Discord Username (optional)

Minkyu Koo#7946

**Comment by tracernz** [comment]
The approach ILS is not tuned until at least in cruise phase and within a certain distance of the destination (300 NM from memory). During departure the departure runway is tuned for runway centreline guidance (RWY mode). To manually tune an ILS you must enter the frequency OR the ident (latter not working in A32NX until #6043 is merged), not both.

**Comment by minkyusimmer** [comment]
> The approach ILS is not tuned until at least in cruise phase and within a certain distance of the destination (300 NM from memory). During departure the departure runway is tuned for runway centreline guidance (RWY mode). To manually tune an ILS you must enter the frequency OR the ident (latter not working in A32NX until #6043 is merged), not both.

I cannot manually enter the ILS frequency. If I select the NAV button then the ILS button, the green light next to the ILS does not illuminate.

**Comment by minkyusimmer** [comment]
> The approach ILS is not tuned until at least in cruise phase and within a certain distance of the destination (300 NM from memory). During departure the departure runway is tuned for runway centreline guidance (RWY mode). To manually tune an ILS you must enter the frequency OR the ident (latter not working in A32NX until #6043 is merged), not both.

Also

**Comment by Benjozork** [comment]
The RMP ILS button is not to be used. It is only there for dual FMGC failures.

Use the RADIO NAV page.

**Comment by tracernz** [comment]
> 
> I cannot manually enter the ILS frequency. If I select the NAV button then the ILS button, the green light next to the ILS does not illuminate.

Are you talking about the Radio Management Panel? That is called remote or backup tuning and shouldn’t be done unless tuning through the MCDU is not possible due to a failure. Manual tuning is done on the RAD NAV page on the MCDU. You can type either a frequency or an ident into the scratchpad then press the LSK next to the item you want to tune. 

> Also, are you saying that it won't tune the approach ILS frequency until 300NM from the destination?

Yes, that is correct.

**Comment by minkyusimmer** [comment]
> The RMP ILS button is not to be used. It is only there for dual FMGC failures.
> 
> Use the RADIO NAV page.

When I try to enter the information through the RADNAV page, it says NOT Allowed.

**Comment by minkyusimmer** [comment]
>  Also, are you saying that it won't tune the approach ILS frequency until 300NM from the destination?
> 
> Yes, that is correct.

For flight that are less than 300NM, what will happen, or what should I do?

**Comment by tracernz** [comment]
> 
> 
> > Also, are you saying that it won't tune the approach ILS frequency until 300NM from the destination?
> > Yes, that is correct.
> 
> For flights that are less than 300NM, what will happen, or what should I do?

It will be automatically tuned when you reach the cruise phase.

**Comment by tracernz** [comment]
> 
> When I try to enter the information through the RADNAV page, it says NOT Allowed.

You will get not allowed if you enter anything other than a frequency (by itself, nothing else)... it should be a format error instead which is fixed up by #6043.
